inst/shiny/modules/tab_nca/nca_results.R

```diff
@@ -102,9 +103,31 @@ nca_results_server <- function(id, pknca_data, res_nca, settings, grouping_vars)
         )
     })
 
+    # Provide the zip file for download
+    output$download_zip <- downloadHandler(
+      filename = function() {
+        project <- session$userData$project_name()
+        datetime <- attr(res_nca(), "provenance")$datetime
+        paste0(project, "_", format(datetime, "%d-%m-%Y"), ".zip")
+      },
+      content = function(fname) {
+        output_tmpdir <- paste0(tempdir(), "/output")
+
+        save_output(output = session$userData$results, output_path = output_tmpdir)
+        files <- list.files(
+          output_tmpdir, full.names = TRUE, pattern = ".[(csv)|(rds)|(xpt)]$", recursive = TRUE
+        )
+        files <- gsub(paste0(output_tmpdir, "/"), "", fixed = TRUE, files)
```

Comment:
issue: We can receive the same result by just settting `full.names` parameter to `FALSE`.
```suggestion
        files <- list.files(output_tmpdir, pattern = ".[(csv)|(rds)|(xpt)]$", recursive = TRUE)
```